src/grid.js

```diff
@@ -18,25 +18,34 @@ const createGrid = (
     rowHeight = null,
     cellAspectRatio = 1,
     pileCellAlignment = 'topLeft',
-    cellPadding = 0
+    cellPadding = 0,
+    cellSize = null
```

Comment:
Instead of adding `cellSize` as another property I suggest replacing `itemSize`. Ultimately, the grid is defined by the cellSize. In `library.js` we can infer `cellSize` from `itemSize` or the other way around. So all the subsequent changes are not necessary. We just need to rename `itemSize` to `cellSize` in `grid.js`.